client/me/purchases/manage-purchase/payment-method-selector/assignment-processor-functions.ts

```diff
@@ -234,6 +235,7 @@ export async function assignNewCardProcessor(
 			city,
 			organization,
 			address,
+			setupKey: String( setupKey ),
```

Comment:
The TypeScript type for `tokenResponse.id` comes from `@stripe/stripe-js` and [says it will always be a string](https://github.com/stripe/stripe-js/blob/d212300e2796954faa7f3b4f916847f31b78be10/types/api/setup-intents.d.ts#L10). Have you found that it's sometimes not a string? It's not a big deal to cast it to a string here but if the type is wrong we should let them know or document it somewhere.